NEWS

```diff
@@ -1,6 +1,27 @@
 Ticket reference tags refer to tickets in the MongoDB JIRA tracking system:
 https://jira.mongodb.org
 
+WiredTiger release 10.0.0, 2020-11-25
+------------------------------------
+
+See the upgrading documentation for details of API and behavior changes.
+
+Significant changes:
+* WT-4427	Make WiredTiger timestamps always on and 8 bytes
```

Comment:
We usually won't highlight timestamp related changes in standalone releases as they are generally not applicable to non-MongoDB users. However, I'm not quite sure about the implication of this ticket, will there be compatiblilty concern for this data format change?